stores/mock.go

```diff
@@ -57,18 +59,23 @@ func (mock *Mockstore) QueryServices(name string) ([]QService, error) {
 	return qServices, nil
 }
 
-func (mock *Mockstore) QueryAuthMethod(service string, host string, typeName string) (map[string]interface{}, error) {
+func (mock *Mockstore) QueryAuthMethods(service string, host string, typeName string) ([]map[string]interface{}, error) {
 
-	var qAuthM = make(map[string]interface{})
+	var qAuthMs []map[string]interface{}
+	var authM map[string]interface{}
 
-	for _, authM := range mock.AuthTypes {
-		qAuthM = utils.StructToMap(authM)
-		if qAuthM["Service"] == service && qAuthM["Host"] == host && qAuthM["Type"] == typeName {
-			return qAuthM, nil
+	if service == "" && host == "" && typeName == "" {
+		return mock.AuthMethods, nil
+	}
+
+	for _, authM = range mock.AuthMethods {
```

Comment:
Do auth_methods need to be pre-instantiated in database before using the service? If so, info on how to minimally set-up them is needed in documentation (A mongo populate script also would be useful). Auth methods contain service + host references. Users can create services by calling the relevant api method. Does this mean that an update in auth_methods collection is needed after service creation?

Reply:
the create auth method functionality will come on the next pr